src/MSBuildLocator/Microsoft.Build.Locator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
how did you pick this set of TFs?

I don't think we should update the .NET Framework target version; we should match what we targeted for VS2017 so that MSBuild is usable in client applications.

For core I'm less sure. Should we target 2.1, the oldest currently-in-support runtime? Should we target net5.0, the newest? should we multitarget?

Reply:
Choosing 5.0 would apparently make it easier, since 2.0 and 5.0 are installed but not 3.1...

Reply:
Personally, I think 3.1 makes sense as in-support, going to stay in support longer than 2.1, and still before net5.0, which I'm guessing is somewhat less commonly used at this point. I'm happy switching to 5.0, though. If we go with that, I assume that also means I should have parts that are just netcoreapp but make the rest of the code run all the time?